entry status main()
{
    lambda<int> multiply_by_two = (int x) => x * 2;
    lambda<int> func = ((int x) => x as int) as lambda<int>;
    lambda<int, int > func2 = (int x, int y) => (int) x * y;
    lambda<int, y, z, ...> x = 3; // will raise an error

    return (status) { .status_code = 0 };
}